plugins/woocommerce-admin/client/customize-store/transitional/services.tsx

```diff
@@ -1,10 +1,9 @@
 /**
  * External dependencies
  */
-import { OPTIONS_STORE_NAME } from '@woocommerce/data';
-import { resolveSelect } from '@wordpress/data';
+import apiFetch from '@wordpress/api-fetch';
 
 export const fetchSurveyCompletedOption = async () =>
-	resolveSelect( OPTIONS_STORE_NAME ).getOption(
-		'woocommerce_admin_customize_store_survey_completed'
-	);
+	await apiFetch( {
```

Comment:
I think this change to CYS is unintended?

Reply:
Thank you for the catch @ilyasfoo 

My bad 🤦 I'll update it soon

Reply:
@ilyasfoo Updated in [Revert back unwanted changes](https://github.com/woocommerce/woocommerce/pull/47915/commits/f8477113c292d1c65c737870c53eac01c18e3471)

[Cache /wc-admin/launch-your-store/survey-completed call](https://github.com/woocommerce/woocommerce/pull/47915/commits/b63e7e713371020d9a69e7a78a2d5339ad6f4dcd) also implements a cache for the call.